docs/blog/2017-09-26-embracing-graphql/index.md

```diff
@@ -2,7 +2,7 @@
 title: "Embracing GraphQL — How I migrated React FAQ to Gatsby v1"
 date: "2017-09-26"
 image: "toa-heftiba-274947.jpg"
-author: "Tim Arney"
+author: "Time Arney"
```

Comment:
In the cherry picking, lost this review change. Should remain "Tim".

Other than that, thank you for separating this out!

Reply:
Oops, thanks again! Re-fixed!